.github/workflows/push.yml

```diff
@@ -0,0 +1,44 @@
+name: Projects check
+
+on:
+  push:
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  projectsbuild:
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - name: kpsr-sdk
+    name: Check ${{ matrix.name }}
+    runs-on: [self-hosted, general, X64]
+    container:
+      image: "registry.klepsydra.io/amd64/basefocal:20.04.0"
```

Comment:
I don't think this has (or should have) access to the registry. There is a public docker image on ghcr.io i think that you should use.

Reply:
@tomasz3dk ?

Reply:
> https://github.com/klepsydra-technologies/kpsr-core-benchmarks/blob/9f627dcb07c634aefe28cbbab7844b6735b2b093/.github/workflows/push.yml#L37

in today's meeting.